dotnet/CoreLib/Configuration/SemanticMemoryConfig.cs

```diff
@@ -87,6 +87,11 @@ public class RetrievalConfig
     /// </summary>
     public string TextGeneratorType { get; set; } = string.Empty;
 
+    /// <summary>
+    /// The ocr generator used to recognize text in images.
```

Comment:
nit: uppercase OCR

Reply:
Happy to confirm to whatever convention you prefer. 

https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions